### PR TITLE
[ANCHOR-865] Add missing columns on ref server start

### PR DESCRIPTION
**Pull request by philipliu**
### Description

This updates the reference server initialization to create new columns in the model but not in the database. It also updates the SELECT query builder to use the new method.

### Context

`customers.bank_name` does not exist, causing SEP-6 transactions to fail.

### Testing

- `./gradlew test`

### Documentation

N/A

### Known limitations

N/A

### CODE REVIEW

**Comment by lijamie98** [comment]
How did the reference server work if the column was missing in local dev?

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt [hunk 1]
```diff
@@ -20,12 +20,17 @@ interface CustomerRepository {
...
+      SchemaUtils.create(Customers)
+      val missingColumnsStatements = SchemaUtils.addMissingColumnsStatements(Customers)
+      missingColumnsStatements.forEach { exec(it) }
+    }
   }
 
   override fun get(id: String): Customer? =
     transaction(db) {
-        Customers.select { Customers.id.eq(id) }
+        Customers.selectAll()
```
Comment (lijamie98) [inline kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt:32]: Why not `SELECT * WHERE id=?`?

This seems inefficient.

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt [hunk 2]
```diff
@@ -99,7 +104,8 @@ class JdbcCustomerRepository(private val db: Database) : CustomerRepository {
         }
       }
     return transaction(db) {
-        Customers.select { query }
+        Customers.selectAll()
```
Comment (lijamie98) [inline kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt:107]: Same as above

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/TransactionKYCRepository.kt [hunk 3]
```diff
@@ -18,12 +18,17 @@ interface TransactionKYCRepository {
 
 class JdbcTransactionKYCRepository(private val db: Database) : TransactionKYCRepository {
   init {
-    transaction(db) { SchemaUtils.create(TransactionKYCs) }
+    transaction(db) {
+      SchemaUtils.create(TransactionKYCs)
+      val missingColumnsStatements = SchemaUtils.addMissingColumnsStatements(TransactionKYCs)
```
Comment (lijamie98) [inline kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/TransactionKYCRepository.kt:23]: Does this happen when we run the newer version of AP on the previous version of the DB?

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt (hunk 1 above)
Comment (philipliu) [reply kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt:32]: The old query builder is deprecated. This is the new way of writing the same query.

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/TransactionKYCRepository.kt (hunk 3 above)
Comment (philipliu) [reply kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/TransactionKYCRepository.kt:23]: Yeah, that's right.

**Comment by philipliu** [comment]
> How did the reference server work if the column was missing in local dev?

The database is recreated frequently in local dev whereas the development has a long-running Postgres instance.

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt (hunk 1 above)
Comment (lijamie98) [reply kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt:32]: The new design of the `exposed` looks weird to me.

**Review by lijamie98 — APPROVED** [review]
LGTM

**Pull request merged**